pynicotine/gtkgui/dialogs/preferences.py

```diff
@@ -1607,6 +1608,16 @@ def __init__(self, application):
             )
         )
 
+        self.buddy_list_position_combobox = ComboBox(
+            container=self.buddy_list_position_label.get_parent(), label=self.buddy_list_position_label,
+            item_selected_callback=self.on_select_buddy_list_position,
+            items=(
+                (_("Separate tab"), "tab"),
+                (_("Chat rooms tab"), "chatrooms"),
```

Comment:
Capitalization `"Chat rooms tab"` -> `"Chat Rooms tab"`

More descriptive choices might be desirable...

"Separate Buddies tab"
"Sidebar in Chat Rooms tab"
"Sidebar always visible in all tabs"

Reply:
Would `Always visible sidebar` work for the last choice?

Reply:
Yes, but for the want of something better.